modules/administration-guide/nav.adoc

```diff
@@ -22,6 +22,7 @@
 *** xref:installing-che-on-red-hat-openshift-local.adoc[]
 *** xref:installing-che-on-minikube.adoc[]
 *** xref:installing-che-on-minikube-keycloak-oidc.adoc[]
+** finding-the-fqdn-of-your-organizations-instance.adoc
```

Comment:
this is triggering because it's plain text. make it an xref and the error goes away.